cmd/helm.go

```diff
@@ -147,15 +147,15 @@ var (
 	// Configurations for granting and revoking access to clients
 	grantedRbacGroupsFlag = cli.StringSliceFlag{
 		Name:  "rbac-group",
-		Usage: "The name of the RBAC group that should be granted access to tiller. Pass in multiple times for multiple groups.",
+		Usage: "The name of the RBAC group that should be granted access to (or revoked from) tiller. Pass in multiple times for multiple groups.",
 	}
 	grantedRbacUsersFlag = cli.StringSliceFlag{
 		Name:  "rbac-user",
-		Usage: "The name of the RBAC user that should be granted access to Tiller. Pass in multiple times for multiple users.",
+		Usage: "The name of the RBAC user that should be granted access to (or revoked from) Tiller. Pass in multiple times for multiple users.",
 	}
 	grantedServiceAccountsFlag = cli.StringSliceFlag{
 		Name:  "rbac-service-account",
-		Usage: "The name and namespace of the ServiceAccount (encoded as NAMESPACE/NAME) that should be granted access to tiller. Pass in multiple times for multiple accounts.",
+		Usage: "The name and namespace of the ServiceAccount (encoded as NAMESPACE/NAME) that should be granted access to (or revoked from) tiller. Pass in multiple times for multiple accounts.",
 	}
```

Comment:
NIT: This probably warrants having new flags so that the help text addresses the command it is associated with. You can refactor the name strings as vars so the command names are reused.

Reply:
To clarify, it is better if this said "revoke" only when getting the help text for revoke, and "grant" only when getting the help text for grant.

Reply:
Will update this.